cmd/webhook/main.go

```diff
@@ -103,8 +103,7 @@ func main() {
 		},
 		Logger: logger,
 	}
-	if err != nil {
-		logger.Fatal("Failed to create the admission controller", zap.Error(err))
+	if err = controller.Run(stopCh); err != nil {
+		logger.Fatal("Failed to start the admission controller", zap.Error(err))
```

Comment:
I think another `Fatalw`?

Was this not being executed before?
Nice catch.

Reply:
See above.

And yes the err was being checked above the Run() so it was looking at the output from the previous time it was set on line 81 making it dead code.